apps/music_proxy.cpp

```diff
@@ -39,13 +39,17 @@
 
 namespace po = boost::program_options;
 
+namespace
+{
+const double defaultMusicTimestep = 0.0001;
+const brion::URI pluginURI( MONSTEER_BRION_SPIKES_PLUGIN_SCHEME + "://" );
```

Comment:
I tried to look around our projects and couldn't identify a clear practice. Since this is the last blocking remark, and in the absence of answers, I am going to merge this code as it is so that we can proceed with the release. We can always correct the style consistently across the project in a later commit if needed.